spec/liquid/liquid_renderer_spec.rb

```diff
@@ -163,13 +163,13 @@
       it 'has the fields from one plugin form' do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name']
+        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name', 'country']
       end
 
       it "checks with the member's liquid data" do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        member = create :member, name: 'Humphrey Bogart', email: 'psycho@killer.com'
+        member = create :member, name: 'Humphrey Bogart', email: 'psycho@killer.com', country: 'US'
```

Comment:
I wouldn't even know where to start debugging that. All I could figure is that when I made `country` not a required field on the default form, it didn't error on this test, and when I made it required, it did.